Content.Server/GameObjects/Components/Disposal/DisposalMailingUnitComponent.cs

```diff
@@ -0,0 +1,847 @@
+            {
+                Logger.WarningS("VitalComponentMissing", $"Disposal unit {Owner.Uid} is missing an anchorable component");
+            }
+
+            if (Owner.TryGetComponent(out CollidableComponent? collidable))
+            {
+                collidable.AnchoredChanged += UpdateVisualState;
+            }
+
+            if (Owner.TryGetComponent(out PowerReceiverComponent? receiver))
+            {
+                receiver.OnPowerStateChanged += PowerStateChanged;
+            }
+
+            UpdateTargetList();
+            UpdateVisualState();
+        }
+
+        public override void OnRemove()
+        {
+            if (Owner.TryGetComponent(out ICollidableComponent? collidable))
+            {
+                collidable.AnchoredChanged -= UpdateVisualState;
+            }
+
+            if (Owner.TryGetComponent(out PowerReceiverComponent? receiver))
+            {
+                receiver.OnPowerStateChanged -= PowerStateChanged;
+            }
+
+            if (_container != null)
+            {
+                foreach (var entity in _container.ContainedEntities.ToArray())
+                {
+                    _container.ForceRemove(entity);
+                }
+            }
+
+            UserInterface?.CloseAll();
+
+            _automaticEngageToken?.Cancel();
+            _automaticEngageToken = null;
+
+            _container = null!;
+
+            base.OnRemove();
+        }
+
+        public override void HandleMessage(ComponentMessage message, IComponent? component)
+        {
+            base.HandleMessage(message, component);
+
+            switch (message)
+            {
+                case RelayMovementEntityMessage msg:
+                    if (!msg.Entity.TryGetComponent(out HandsComponent? hands) ||
+                        hands.Count == 0 ||
+                        _gameTiming.CurTime < _lastExitAttempt + ExitAttemptDelay)
+                    {
+                        break;
+                    }
+
+                    _lastExitAttempt = _gameTiming.CurTime;
+                    Remove(msg.Entity);
+                    break;
+            }
+        }
+
+        private void OnReceiveNetMessage(int frequency, string sender, IReadOnlyDictionary<string, string> payload, object _, bool broadcast)
+        {
+            if (payload.TryGetValue("command", out var command) && Powered)
+            {
+                if (command == "mailer_tag" && payload.TryGetValue("tag", out var tag))
+                {
+                    _targetList.Add(tag);
+                    UpdateInterface(false);
+                }
+
+                if (command == "get_mailer_tag")
+                {
+                    if (_tag == "" || !Powered)
+                        return;
+
+                    var data = new Dictionary<string, string>
+                    {
+                        {"command", "mailer_tag"},
+                        {"tag", _tag}
```

Comment:
Usages of strings like these should be replaced by constants defined somewhere else, if you want to use a dictionary instead of something else.